Ginger/Ginger/ALM/ALMConnectionPage.xaml.cs

```diff
@@ -521,5 +537,29 @@ private void HandleLinkClick(object sender, RequestNavigateEventArgs e)
             System.Diagnostics.Process.Start(@"http://ginger/Downloads/Other");
             e.Handled = true;
         }
+
+        private void RestAPICheckBox_Checked(object sender, RoutedEventArgs e)
+        {
+            if (App.UserProfile.Solution.AlmType == ALMIntegration.eALMType.QC)
+            {
+                App.UserProfile.Solution.UseRest = true;
+                ExampleURLHint.Content = "Example: http://server:8080/";
+                if (ServerURLTextBox.Text.EndsWith("qcbin"))
+                    ServerURLTextBox.Text = ServerURLTextBox.Text.Replace("qcbin", "");
+            }
+            else
+            {
+                Reporter.ToUser(eUserMsgKeys.StaticInfoMessage, "Rest connection refering only to QC");
+                RestAPICheckBox.IsChecked = false;
+            }
+        }
+
```

Comment:
Why not to hide the Checkbox in case the QC radio button is not selected in stead letting the user check it and show message that it is not relevant to RQM?
also this URL formating shouldn't be here because use can set/change the URL before/after clicking on the check box, it should be changed when user actually tries to connect